introduction.md

```diff
@@ -1 +1,5 @@
-Hi, I'm Louis Fyne. I'm 6'3", and maintain a very consistent panda bear shape. My fave path is JavaScript, of course!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?